apps/src/templates/levelSummary/SummaryResponses.jsx

```diff
@@ -23,21 +20,13 @@ const CheckForUnderstanding = ({
   levels,
 }) => {
   const currentLevel = levels.find(l => l.activeId === currentLevelId);
-  const nextLevel = levels.find(l => l.position === currentLevel.position + 1);
-  const sectionParam = selectedSection?.id
-    ? `?section_id=${selectedSection.id}`
-    : '';
 
   // To avoid confusion, if a teacher tries to view the summary as a student,
   // send them back to the level in Participant mode instead.
   if (viewAs === ViewType.Participant) {
     document.location.replace(currentLevel.url + document.location.search);
   }
```

Comment:
a little strange to have this here in the responses component, but i wasn't sure where else to put something like this that isn't tied to any one visual element, but causes a side-effect on the page